tests/integration/simple_copy_test/test_simple_copy.py

```diff
@@ -308,4 +308,4 @@ def test__snowflake__incremental_merge_columns(self):
             }
         })
         self.seed_and_run()
-        self.assertTablesEqual("incremental_update_cols", "expected_result")
+        self.assertTablesEqual("incremental_update_cols", "expected_result")
```

Comment:
this change might not be needed in this PR, worth excluding since it isn't related.

Reply:
Yeah it's just some whitespace that got nixed by black.